iOS/Flows/Conversation/Threads/ThreadViewController.swift

```diff
@@ -169,127 +171,44 @@ class ThreadViewController: DiffableCollectionViewController<MessageSequenceSect
     }
 
     override func getAnimationCycle() -> AnimationCycle? {
-        var cycle = super.getAnimationCycle()
-        cycle?.shouldConcatenate = false
-        // Scroll to the lastest reply.
-        if let threadLayout = self.collectionView.collectionViewLayout as? ThreadCollectionViewLayout {
-            let lastReplyIndex = clamp(self.messageController.replies.count - 1, min: 0)
-            let yOffset = CGFloat(lastReplyIndex) * threadLayout.itemHeight
-            cycle?.scrollToOffset = CGPoint(x: 0, y: yOffset)
-        }
-
-        return cycle
+        return nil
     }
+}
 
-    // MARK: - SwipeableInputAccessoryViewDelegate
-
-    var prepareForSend = false
-
-    func swipeableInputAccessory(_ view: SwipeableInputAccessoryView, swipeIsEnabled isEnabled: Bool) {
```

Comment:
Similar to the ConvoList VC, all the red here has been moved to the delegate object.